app/views/docs/permissions.phtml

```diff
@@ -44,60 +44,62 @@
     </thead>
     <tbody>
     <tr>
-        <td>
-            role:all
-            <p class="margin-top-small text-fade">version >= 0.12</p>
-        </td>
+        <td>Role.any()</td>
         <td>Wildcard permission. Gives anyone read or write access.</td>
```

Comment:
We specify "read or write", tho we now also have update. Can we mention that this gives you relevant permission depending on where you use it? Or maybe not mention at all, just say who will be allowed with this permission?

Reply:
Maybe `Wildcard permission that gives access to anyone.`